partitura/musicanalysis/performance_codec.py

```diff
@@ -59,6 +61,9 @@ def encode_performance(
         List of unique onset ids. Returned only when return_u_onset_idx
         is set to True.
     """
+    if isinstance(part, Score):
+        # TODO change note ids in alignment
```

Comment:
Is this TODO open?

Reply:
I am unsure, if somebody inputs a score with ids then the renaming scheme of merge parts will rename id `n01` of Part 0 to `P00-n01`. It has to stay consistent with the alignment file, I guess this is a rare occasion where the alignment refers to note ids in one way but the score is written in a different way. For example, if the score has ids `n01` in Part 0 and `n02` in part one same has alignment but after merging parts this would change.

Reply:
This has a better fix in PR #233